src/components/shared/OutlinedToggleButton.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+    SxProps,
+    Theme,
+    ToggleButton,
+    ToggleButtonProps,
+    toggleButtonClasses,
+    useTheme,
+} from '@mui/material';
+import {
+    intensifiedOutline,
+    outlinedButtonBackground,
+    primaryColoredOutline,
+    primaryColoredOutline_hovered,
+} from 'context/Theme';
+
+interface Props extends ToggleButtonProps {
+    defaultStateSx?: SxProps<Theme>;
+    disabledStateSx?: SxProps<Theme>;
+    selectedStateSx?: SxProps<Theme>;
+}
+
+function OutlinedToggleButton({
+    children,
+    defaultStateSx,
+    disabled,
+    disabledStateSx,
+    selectedStateSx,
+    selected,
+    value,
+    onChange,
+    onClick,
+}: Props) {
+    const theme = useTheme();
+
+    let sx: SxProps<Theme> = {
+        px: '9px',
+        py: '3px',
+        border: intensifiedOutline[theme.palette.mode],
+        borderRadius: 2,
+        [`&.${toggleButtonClasses.selected}`]: selectedStateSx ?? {
+            'backgroundColor': outlinedButtonBackground[theme.palette.mode],
+            'border': primaryColoredOutline[theme.palette.mode],
+            'color': theme.palette.primary.main,
+            '&:hover': {
+                border: primaryColoredOutline_hovered[theme.palette.mode],
+            },
+        },
+    };
+
+    if (defaultStateSx) {
+        sx = { ...sx, ...defaultStateSx };
+    }
+
+    if (disabledStateSx) {
+        sx = { ...sx, [`&.${toggleButtonClasses.disabled}`]: disabledStateSx };
+    }
```

Comment:
This is fine for now - but we need to handle this a bit better by just consuming the `sx` that can be passed in.